### PR TITLE
Import all handlers at once

### PR DESCRIPTION
**Pull request by Androz2091**
### Items Changed

- [ ] Commands
- [ ] Events
- [x] Handlers
- [ ] Structures
- [ ] Tasks
- [ ] Documentation
- [ ] Other: ______

### Description

Use 1 import instead of 9 (like https://github.com/dilame/instagram-private-api/tree/master/src/types)

### Have You...?

- [x] I have read the [Contributing Guidelines](https://github.com/typicalbot/typicalbot/blob/master/.github/CONTRIBUTING.md).
- [x] I have completed the [pull request template](https://github.com/typicalbot/typicalbot/blob/master/.github/PULL_REQUEST_TEMPLATE.md).
- [x] I have checked open [pull requests](https://github.com/typicalbot/typicalbot/pulls) for upcoming features and fixes.

### CODE REVIEW

**Review by nsylke — APPROVED** [review]
LGTM

File: src/handlers/index.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export * from './Commands';
+export * from './Database';
+export * from './Events';
+export * from './Functions';
+export * from './ModerationLogs';
```
Comment (tjrgg) [inline src/handlers/index.ts]:
```suggestion
export * from './ModerationLog';
```

Comment (Androz2091) [reply src/handlers/index.ts]: My bad, fixed

**Comment by nsylke** [comment]
@Skillz4Killz approve or request changes please

**Comment by Skillz4Killz** [comment]
I do prefer the way you are doing it and I think this is a great PR, but at first glance I don't think this will run? Did you try running or building bot with these changes? @Androz2091 

I think you might have the change the way we are exporting on each of those files. Can someone try and build this I can't access this today sadly.

EDIT: Nevermind, it looks like the Github Actions is building it already and it is indeed failing. 

1. Id love to see if you can patch all the eslint errors. `npm run lint:fix`
2. The TSC errors are where the build is failing.

**Comment by tjrgg** [comment]
@Skillz4Killz The GitHub action build fails.

**Comment by Androz2091** [comment]
Sorry, I couldn't test it (as you can see, I made a mistake in a filename by the way, sorry). But I use it for some of my projects, like here: https://github.com/Androz2091/discord-backup (types folder) and it works perfectly

**Comment by nsylke** [comment]
I've moved away from using index files as an export, unless its a library. Thus, I don't think we need make this change. An alternative would be to use module aliases (which I use in React projects).

**Pull request closed without merging**